unifi_protect_backup/unifi_protect_backup.py

```diff
@@ -289,6 +295,52 @@ async def rclone_purge_old():
                 logger.warn(f"stdout:\n{stdout.decode()}")
                 logger.warn(f"stderr:\n{stderr.decode()}")
 
+        # We need to catch websocket disconnect and trigger a reconnect.
+        @aiocron.crontab("* * * * *")
+        async def check_websocket_and_reconnect():
+            logger.debug("Checking the status of the websocket...")
+            if self._protect.check_ws():
+                logger.debug("OK")
```

Comment:
I would make this a little more explicit in what is OK because it could get interrupted by other messages

Reply:
Valid point. Made more explicit.